walk-in-interview/src/main/webapp/new-job/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="../style.css">
+</head>
+
+<body>
+
+    <div class="header-container">
+        <input id="new-job-cancel">
+        <div id="new-job-page-title"></div>
+        <input id="new-job-submit">
```

Comment:
maybe u can add a comment to say that the content will be dynamically rendered in other js file